### DISCUSSION TITLE
New risk measure

### DISCUSSION THREAD

**Discussion by Nicky9529** [dcajasn/Riskfolio-Lib#47] [Ideas]
Thanks for your great package. Could you add the Serenity Ratio as presented by Keyquant to the list used, say, for the HERC functionnality?

https://www.keyquant.com/Download/GetFile?Filename=%5CPublications%5CKeyQuant_WhitePaper_APT_Part1.pdf

**Comment by dcajasn** [comment]
Some time ago, someone ask if I can add the serenity ratio and smart beta from keyquant, but both models seems to be not convex and the authors don't reveal how they optimize both models. For this reasons, I don't plan to add serenity risk and smart beta. Here is the reference [link](https://github.com/dcajasn/Riskfolio-Lib/issues/25).

**Reply by Nicky9529** [reply]
Ok, so no viable convex solutions there. Thanks for your reply.
